test/engines/wintermute/path_utils.h

```diff
@@ -53,14 +53,10 @@ class PathUtilTestSuite : public CxxTest::TestSuite {
 				Wintermute::PathUtil::getDirectoryName(windowsCapPath),
 				Common::String("C:\\SOME\\")
 				);
-
```

Comment:
Preferably remove these from the preceding commit, and just introduce them here.